spotify_tensorflow/tfx/tft.py

```diff
@@ -71,10 +70,6 @@ def run(self, args=None):
             "--transform_fn_dir",
             required=False,
             help="path to the saved transform function")
-        parser.add_argument(
-            "--requirements_file",
```

Comment:
Do we need to replace this with `--setup_file`, or does that get passed elsewhere?

Reply:
nope, but I'll add a `--setup_file` option in luigi base task, so users can set it directly in their job.